proto/rdp/README.md

```diff
@@ -0,0 +1,286 @@
+---
+title: Reviewdog Diagnostic Protocol (RDP)
+date: 2020-06-15
+author: haya14busa
+status: Proposed / Experimental
+---
+
+# Status
+
+This document proposes Reviewdog Diagnostic Protocol (RDP) and it's still
+in experimental stage.
+
+Any review, suggestion, feedback, criticism, and comments from anyone is very
+much welcome. Please leave comments in Pull Request ([#629](https://github.com/reviewdog/reviewdog/pull/629))
+, in issue [#628](https://github.com/reviewdog/reviewdog/issues/628) or
+file [an issue](https://github.com/reviewdog/reviewdog/issues).
+
+The document and the actual definition are currently in under
+https://github.com/reviewdog/reviewdog repository, but we may create a separate
+repository once it's reviewed and stabilized.
+
+# Reviewdog Diagnostic Protocol (RDP)
```

Comment:
The abbreviation `RDP` will create ambiguity with the popular `Remote Desktop Protocol`.
I think it would be better to find another abbreviation.

Reply:
True. Do you have any suggestions? 
I wonder whether *protocol* is good or not as well. https://github.com/reviewdog/reviewdog/tree/rdp/proto/rdp#open-questions

But if we use *Format* instead of *Protocol*, **RDF** (*Reviewdog Diagnostic Format*) is also used by [Resource Description Framework](https://www.w3.org/RDF/) as well.

The acronym is basically for providing short format name (currently rdpjsonl / rdpjson), so maybe it's ok even if there are other usages. Any thought?

Reply:
I'd rather not collide w/ RDF / RDP -- I use both.

The lazy answer would be RDDP or RDDF.